autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -135,13 +135,14 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleState(Cluster clus
 
     private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Cluster cluster,
             DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
-
-        alertValidator.validateEntitlementAndDisableIfNotEntitled(cluster);
-        alertValidator.validateDistroXAutoscaleClusterRequest(cluster, autoscaleClusterRequest);
-
         if (Boolean.TRUE.equals(autoscaleClusterRequest.getUseStopStartMechanism())) {
             alertValidator.validateStopStartEntitlementAndDisableIfNotEntitled(cluster);
         }
+
+        Boolean enableStopStartScaling = asClusterCommonService.shouldEnableStopStartBasedOnEntitlement(cluster,
+                autoscaleClusterRequest.getTimeAlertRequests().size(), autoscaleClusterRequest.getUseStopStartMechanism());
```

Comment:
This can be cleaned up by passing the `AutoscaleClusterRequest` object as an argument directly, instead of explicitly passing the size of timeAlertRequests and an additional boolean

Reply:
Done